daemon/api_sideload_n_try.go

```diff
@@ -218,39 +219,157 @@ func sideloadOrTrySnap(ctx context.Context, c *Command, body io.ReadCloser, boun
 
 // sideloadedInfo contains information from a bunch of sideloaded snaps
 type sideloadedInfo struct {
-	sideInfos                  []*snap.SideInfo
-	names, origPaths, tmpPaths []string
+	snaps      []sideloadSnapInfo
+	components []sideloadComponentInfo
 }
```

Comment:
Possibly worth adding a comment to the struct members, I see that sideloadComponentInfo is used when the snap is not being installed at the same time but it would be good to make it more clear.